lib/src/main/cpp/examples/GetDepthExample.cpp

```diff
@@ -0,0 +1,29 @@
+#include <string>
+#include <fstream>
+#include <vector>
+#include <iostream>
+#include "../include/DepthEstimator.h"
+
+const int kCliArgsNum = 4;
+
+inline std::vector<char> streamToVector(std::ifstream &s) {
+    return {std::istreambuf_iterator<char>(s), std::istreambuf_iterator<char>()};
+}
+
+int main(int argc, char *argv[]) {
+    if (argc != kCliArgsNum) {
+        std::cerr << "Illegal CLI arguments num. Expected " << kCliArgsNum << "(including the command name) but was "
+                  << argc;
+        return 1;
+    }
+
+    auto calibPath = argv[1], leftImgPath = argv[2], rightImgPath = argv[3];
+
+    std::ifstream lImgStream(leftImgPath), rImgStream(rightImgPath);
+    auto lImg = streamToVector(lImgStream), rImg = streamToVector(rImgStream);
+
+    DepthEstimator depthEstimator(calibPath);
+    std::vector<float> depth = depthEstimator.estimateDepth(lImg, rImg);
```

Comment:
I thought it's better to leave it as is because this is an example and `estimateDepth` function name doesn't give much information about its return type.